identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/listner/AppPortalRoleManagementListener.java

```diff
@@ -20,6 +20,8 @@
 
 import org.apache.commons.lang.StringUtils;
 import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.identity.organization.management.service.exception.OrganizationManagementException;
```

Comment:
update the the license with year range for editing files.
It should be 

Copyright (c) 2023-2024, WSO2 LLC. (http://www.wso2.com).

Reply:
This is now updated.